steps/bitrise-step-activate-bazel-cache/step-info.yml

```
@@ -0,0 +1 @@
+maintainer: community
```

```suggestion
maintainer: bitrise
```

Once you commit this change, the CI will fail and it will require a manually approved restart, I can help